src/MSBuild.UnitTests/XMake_Tests.cs

```diff
@@ -2711,6 +2711,13 @@ public void EndToEndMinimumMessageImportance(string arguments, MessageImportance
 
             TransientTestProjectWithFiles testProject = testEnvironment.CreateTestProjectWithFiles(projectContents);
 
+            // If /bl is specified, set a path for the binlog that is defined by the test environment
+            if (arguments.Contains("/bl"))
```

Comment:
We know exactly what the possible values for arguments are, and none of them seem to have /bl. What does adding this check accomplish?

Reply:
https://github.com/JaynieBai/msbuild/blob/ac9aad106dba109556bc7295c8660f068c225825/src/MSBuild.UnitTests/XMake_Tests.cs#L2681-L2685

Reply:
Oh, you're right; I just didn't scroll up enough 😅